app/controllers/shared/api.php

```diff
@@ -76,8 +76,8 @@
             ;
         }
 
-        if (($abuse->check() // Route is rate-limited
-        && App::getEnv('_APP_OPTIONS_ABUSE', 'enabled') !== 'disabled') // Abuse is not disabled
+        if ((App::getEnv('_APP_OPTIONS_ABUSE', 'enabled') !== 'disabled' // Route is rate-limited
```

Comment:
Why did we change this condition?

Reply:
The abuse check was still technically running the DB calls even if the _APP_OPTIONS_ABUSE environment variable was set by placing $abuse->check() after the env check we prevent that from happening

Reply:
For performance, basically it will not write to the abuse table if abuse is disabled.